### PR TITLE
RONDB-240: API server fixes for MacOS

### PR DESCRIPTION
**Pull request by smkniazi**

### CODE REVIEW

File: mysql-test/suite/rdrs/include/run_rdrs_unit_test.inc [hunk 1]
```diff
@@ -30,7 +30,7 @@ if (!$res)
 }
 
 
-my $cmd="LD_LIBRARY_PATH=$ENV{LD_LIBRARY_PATH}; CGO_LDFLAGS=$ENV{CGO_LDFLAGS} $rdrs_test_cmd";
+my $cmd="export DYLD_LIBRARY_PATH=$ENV{LD_LIBRARY_PATH}; export LD_LIBRARY_PATH=$ENV{LD_LIBRARY_PATH}; export CGO_LDFLAGS=$ENV{CGO_LDFLAGS}; $rdrs_test_cmd";
```
Comment (mronstro) [inline mysql-test/suite/rdrs/include/run_rdrs_unit_test.inc:33]: DYLDB_LIBRARY_PATH=LD_LIBRARY_PATH in first export, need fix

File: storage/ndb/rest-server/rest-api-server/Makefile [hunk 2]
```diff
@@ -41,6 +38,8 @@ BUILD_COMMAND=go build -ldflags="-w \
      -o ${OUTPUT_DIR}/rdrs ./cmd/server/main.go
 
 rdrs:
+	export DYLD_LIBRARY_PATH=$DYLD_LIBRARY_PATH:${RDRS_LIB_DIR} && \
+	export LD_LIBRARY_PATH=$DYLD_LIBRARY_PATH:${RDRS_LIB_DIR} && \
```
Comment (mronstro) [inline storage/ndb/rest-server/rest-api-server/Makefile]: LD=DYLD needs fixing

File: storage/ndb/rest-server/rest-api-server/Makefile [hunk 3]
```diff
@@ -61,7 +60,11 @@ build_proto:
 test:
 	@go clean -testcache
 	@echo "Getting cross-package (integration) coverage and writing it to $(COVERAGE_OUTPUT_FILE)"
-	CGO_LDFLAGS="-g -O3 -L${RDRS_LIB_DIR} -lrdrclient" RDRS_CONFIG_FILE=$(RDRS_CONFIG_FILE) go test -p 1 -coverprofile $(COVERAGE_OUTPUT_FILE) -coverpkg ./... `go list ./... | grep -v cmd`
+	export DYLD_LIBRARY_PATH=$DYLD_LIBRARY_PATH:${RDRS_LIB_DIR} && \
+	export LD_LIBRARY_PATH=$DYLD_LIBRARY_PATH:${RDRS_LIB_DIR} && \
```
Comment (mronstro) [inline storage/ndb/rest-server/rest-api-server/Makefile]: LD=DYLD needs fixing

**Review by mronstro — APPROVED** [review]
Approved after fixing LD=DYLD and DYLD=LD problems

File: storage/ndb/rest-server/rest-api-server/Makefile (hunk 2 above)
Comment (smkniazi) [reply storage/ndb/rest-server/rest-api-server/Makefile]: fixed

File: storage/ndb/rest-server/rest-api-server/Makefile (hunk 3 above)
Comment (smkniazi) [reply storage/ndb/rest-server/rest-api-server/Makefile]: fixed

File: mysql-test/suite/rdrs/include/run_rdrs_unit_test.inc (hunk 1 above)
Comment (smkniazi) [reply mysql-test/suite/rdrs/include/run_rdrs_unit_test.inc:33]: this is correct

**Pull request merged**